cip-0026.md

```diff
@@ -10,7 +10,7 @@
 
 From the very beginning, creating accounts on Burstcoin has not been entirely intuitive, and the process suffers from the chicken-and-egg problem:
 For an account to be stored on the blockchain, there must be at least one transaction involving the named account.
-However, a problem arises from the theoretical possibility of a collision in Account ID or the Reed-Solomon representation, since the address space is smaller than that of the public key. Therefore, an account without a link to the public key is considered insecure while inactive.
+However, a problem arises from the theoretical possibility of a collision in Account ID or the Reed-Solomon representation, since the address space for all passoble addresses is smaller than that of the public key. Therefore, an account without a link to the public key is considered insecure while inactive.
```

Comment:
OK, just adjust "passoble" to "possible".

Reply:
ok,ty